Riot/Modules/MatrixKit/Categories/NSAttributedString+MatrixKit.swift

```diff
@@ -29,4 +29,11 @@ public extension NSAttributedString {
         
         return unquotedSubstrings.joined(separator: " ") as NSString
     }
+
+    /// Returns a new attributed string by removing all links from the receiver.
+    @objc var byRemovingLinks: NSAttributedString {
```

Comment:
Do not hesitate to move this extension in the app folder if possible and use `vc_byRemovingLinks` prefix to avoid potential conflicts with Apple API or other lib extensions.

Reply:
Thanks for the suggestion, updated.